catalog/views.py

```diff
@@ -1,3 +1,27 @@
 from django.shortcuts import render
 
 # Create your views here.
+
+from catalog.models import Book, Author, BookInstance, Genre
+
+
+def index(request):
+    """View function for home page of site."""
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
```

Comment:
bỏ all dư thừa cho các line
```suggestion
    num_books = Book.objects.count()
```